com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/Templates/Terrain/BasemapGenShaderPass.template

```diff
@@ -0,0 +1,339 @@
+Pass
```

Comment:
This template may be able to be shared between URP and HDRP. They will likely still need to implement separate BasemapGenVertex.template.hlsl files, since HDRP and URP have different function names which are expected for their shader pipelines. But it would be ideal to share one template for what basically amounts to a blit shader.

Alternatively, if ShaderGraph is planning on adding support for Blit shaders in some other way, it'd be best to align with that. However, Terrain has some specific needs for its base passes that can be reviewed at SplatMaterialData::UpdateBaseMaps